recovery/mainwindow.cpp

```diff
@@ -951,29 +954,48 @@ void MainWindow::on_list_doubleClicked(const QModelIndex &index)
     }
 }
 
-void MainWindow::startNetworking()
+void MainWindow::copyWpa()
 {
+    //This file is the one used by dhcpcd
     QFile f("/settings/wpa_supplicant.conf");
-
     if ( f.exists() && f.size() == 0 )
     {
         /* Remove corrupt file */
         f.remove();
     }
-    if ( !f.exists() )
+
+    /* If user supplied a wpa_supplicant.conf on the FAT partition copy that one to settings regardless */
+    if (QFile::exists("/mnt/wpa_supplicant.conf"))
     {
-        /* If user supplied a wpa_supplicant.conf on the FAT partition copy that one to settings
-           otherwise copy the default one stored in the initramfs */
-        if (QFile::exists("/mnt/wpa_supplicant.conf"))
-            QFile::copy("/mnt/wpa_supplicant.conf", "/settings/wpa_supplicant.conf");
-        else
-        {
-            qDebug() << "Copying /etc/wpa_supplicant.conf to /settings/wpa_supplicant.conf";
-            QFile::copy("/etc/wpa_supplicant.conf", "/settings/wpa_supplicant.conf");
-        }
+        qDebug() << "Copying  user wpa_supplicant.conf to /settings/wpa_supplicant.conf";
```

Comment:
Two spaces between "Copying" and "user" ;)

Reply:
Eagle-eyed! Yes.